version.yaml

```diff
@@ -0,0 +1,24 @@
+# compatible kiali version range with istio version
```

Comment:
I do not like the name of this file. `version.yaml` implies this file contains the version of Kiali, which is not what it is. Rename this file to something more descriptive of what it is - something like `version-compatibility-matrix.yaml`.

Reply:
Sure, I will update this name later.

Reply:
Done